frontend/public/components/storage-class-form.tsx

```diff
@@ -104,39 +106,39 @@ export class StorageClassForm_ extends React.Component<
   CSIStorageTypes = Object.freeze({
     ...this.getExtensionsStorageClassProvisioners(Provisioner.CSI),
     'ebs.csi.aws.com': {
-      title: 'AWS CSI',
+      title: 'AWS CSI', // t('storage-class-form~AWS CSI')
       provisioner: 'ebs.csi.aws.com',
       allowVolumeExpansion: true,
       parameters: {
         type: {
-          name: 'Type',
+          name: 'Type', // t('storage-class-form~Type')
           values: { gp2: 'gp2', io1: 'io1', sc1: 'sc1', st1: 'st1', standard: 'standard' },
           hintText: 'Select AWS Type. Default is gp2',
         },
         iopsPerGB: {
-          name: 'IOPS Per GiB',
-          hintText: 'I/O operations per second per GiB',
+          name: 'IOPS Per GiB', // t('storage-class-form~IOPS Per GiB')
```

Comment:
```suggestion
          name: 'IOPS per GiB', // t('storage-class-form~IOPS per GiB')
```

Reply:
@sg00dwin It looks like most of my comments in this file are still pending.

Reply:
updated